luminaire/model/window_density.py

```diff
@@ -578,20 +606,24 @@ def _get_result(self, input_df=None, detrend_order=None, agg_data_model=None, va
             # based test for the past standard deviations to detect anomalies
             elif baseline_type == "aggregated":
                 baseline_sds = np.array(baseline).std(1).tolist()
-                baseline_execution_data = copy.copy(baseline)
-                baseline_execution_data.append(adjusted_execution_data)
-                pca = PCA()
-                scores = pca.fit_transform(StandardScaler().fit_transform(baseline_execution_data))
-                robust_cov = MinCovDet().fit(scores[:, :3])
-                mahalanobis_distance = robust_cov.mahalanobis(scores[:, :3])
-                pvalue_mahalanobis = 1 - st.chi2.cdf(mahalanobis_distance[-1],
-                                                     np.array(baseline_execution_data).shape[1])
-
-                gamma_alpha, gamma_loc, gamma_beta = st.gamma.fit(baseline_sds)
-                pvalue_gamma = 1 - st.gamma.cdf(np.std(adjusted_execution_data), gamma_alpha, gamma_loc, gamma_beta)
-                if pvalue_mahalanobis < self.sig_level or pvalue_gamma < self.sig_level:
-                    is_anomaly = True
-                prob_of_anomaly = 1 - min(pvalue_mahalanobis, pvalue_gamma)
+                if detrend_order == 0:
+                    adjusted_execution_data = [adjusted_execution_data]
```

Comment:
`adjusted_execution_data` is already an list. Why is this `if` block needed? Also, does it need an `else`?

Reply:
When detrend_order is 0, we are not doing any adjustments in the execution data. This step makes sure we don't need to handle this separately when we run the following for loop.